src/styles/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  width: 198px;
+  background-color: white;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  font-family: $roboto-font;
+  color: $text-color;
+  font-weight: 400;
+  transition: all 300ms linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 16px;
+  }
+
+  &__name {
+    margin-top: 40px;
+  }
+
+  &__title {
+    font-size: $headline-size;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0;
+    color: $text-color;
+    transition: all 300ms linear;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+    transition: all 300ms linear;
+  }
+
+  &__product-code {
+    margin: 0;
+    font-size: $subscript-size;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-top: 4px;
+  }
+
+  &__review-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__review-text {
+    font-size: $subscript-size;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: $headline-size;
+    line-height: 18px;
+    color: $secondary-color;
+    margin: 0;
+  }
+
+  &__price {
+    font-size: $price-text;
+    line-height: 18px;
+    font-weight: 700;
+    margin: 0;
+  }
+
+  &__image {
+    width: 100%;
+    height: 134px;
+    padding: 0;
+    margin-top: 16px;
+    position: relative;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    height: 100%;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 38px;
+    background-color: $accent-color;
+    font-size: $text-size;
+    font-weight: 700;
+    border: 1px solid $accent-color;
+    border-radius: 5px;
+    margin-top: 16px;
+    transition: background-color 300ms linear;
```

Comment:
You have a 'transition: background-color 300ms linear;' property set on the .card &__button class. But you have already set the transition property in the .card class. There's no need to repeat the transition property in the .card &__button class. Remove it to keep your code DRY (Don't Repeat Yourself).